Libraries/Components/ScrollResponder.js

```diff
@@ -471,6 +472,15 @@ var ScrollResponderMixin = {
    * @platform ios
    */
   scrollResponderFlashScrollIndicators: function() {
+    if (Platform.OS === 'android') {
```

Comment:
Hmm! In honesty, I wasn't quite sure why on iOS there was an explicit ScrollViewManager as a native module, but on android there isn't one

Unfortunately I'm lacking the iOS-foo to test this change out, and I can't see any equivalent docs for running a modified of react-native with iOS - similar to the the [Android Building React Native from source](https://facebook.github.io/react-native/docs/android-building-from-source.html) docs. I'm going to have to punt on this one unfortunately!